### DISCUSSION TITLE
When adding a URL to add: SyntaxError: Unexpected token '<', "

### DISCUSSION THREAD

**Discussion by KamillaPup** [Difegue/LANraragi#875] [Q&A] [answered]
When adding an archive URL it breaks with the error in the toast notifier `SyntaxError: Unexpected token '<', "`

```
upload:1 Unchecked runtime.lastError: A listener indicated an asynchronous response by returning true, but the message channel closed before a response was received
api/download_url:1     Failed to load resource: the server responded with a status of 500 ()
```

This also happens with the extension. I have LANraragi running in a docker in Plesk and forwarding the port to a subdomain so I can run it on 443 and have an SSL.

I remember reading something that the port forwarding screws up the connection between the docker container and the target website.

What can I do to resolve this?

**Comment by KamillaPup** [comment]
So I found out this was a permissions issue with the database, but upon fixing that, now it just spins and spins under downloading. I tested with several files, the most recent only being 12 MB and it shouldn't take this long. What else can I try?

**Comment by KamillaPup** [comment, marked as answer]
After fixing permissions and rebooting, it works fine! Silly me…

**Reply by Difegue** [reply]
Good job fixing it ! Sorry for not getting to this in time to help 😅
